modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-6-0_{context}"]
+= Release notes for {gitops-title} 1.6.0
+
+{gitops-title} 1.6.0 is now available on {product-title} 4.8, 4.9, and 4.10.
+
+[id="new-features-1-6-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, `ApplicationSet` is now available in GA and is merged to Argo CD core. link:https://issues.redhat.com/browse/GITOPS-1958[GITOPS-1958]
+
+* With this update, the latest releases of the {gitops-title} will only be available in **latest** and version based channels like, **gitops-1.6**. Users existing on the **stable** channel should move to either the **latest** or the version based channel for getting upgrades. link:https://issues.redhat.com/browse/GITOPS-1791[GITOPS-1791]
+
+* With this update, the parameters of `spec.sso` field that controlled the keycloak configurations are moved to `.spec.sso.keycloak`.
+The parameters of `.spec.dex` field are moved to `.spec.sso.dex`. Dex can now be enabled/disabled using `.spec.sso.provider`. The existing configuration methods will be supported alongside the new configuration methods until GitOps v1.9, when support for `.spec.dex`, `DISABLE_DEX` and `.spec.sso` fields for keycloak configuration will be removed from the operator link:https://issues.redhat.com/browse/GITOPS-1983[GITOPS-1983]
+
+* With this update, the Argo CD Notifications Controller is available as an optional workload that can be enabled/disabled through `.spec.notifications.enabled` parameter in the Argo CD custom resource. The Argo CD Notifications Controller is available as Technical Preview. link:https://issues.redhat.com/browse/GITOPS-1917[GITOPS-1917]
+
+* With this update, the default resource exclusions are added to the `csv` for Tekton pipeline runs and task runs. Argo CD, by default prunes these resources. These resource exclusions are added to the new Argo CD instances that are created from the *OpenShift console*. If the instances are created from the CLI the resources are not added. link:https://issues.redhat.com/browse/GITOPS-1876[GITOPS-1876]
+
+* With this update, you can set the value of the `resourceTrackingMethod` to select the tracking method used by Argo CD. link:https://issues.redhat.com/browse/GITOPS-1862[GITOPS-1862]
```

Comment:
- Label `resourceTrackingMethod`. Is it a parameter or object?: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references
- I don't see it documented in previous versions of openshift-docs. Is this a new feature? So `resourceTrackingMethod` is new, not just the ability to select the tracking method. If so, how about:
```suggestion
* With this update, you can select the tracking method that by Argo CD uses by setting the `resourceTrackingMethod` parameter in <the location of this parameter>. link:https://issues.redhat.com/browse/GITOPS-1862[GITOPS-1862]
```